test/index.js

```diff
@@ -20,6 +22,20 @@ const walletTwoAddresses = [
   '0xbbafcf3d00fb625b65bb1497c94bf42c1f4b3e78',
   '0x49dd2653f38f75d40fdbd51e83b9c9724c87f7eb',
 ];
+
+// Shims for globals that don't exist in test environment
+global.crypto = {
+  getRandomValues: () => {
+    return Math.floor(Math.random(1000000));
+  },
+  subtle: {
+    digest: (algorithm, data) =>
+      new Uint8Array(Date.now().toString().split('')).buffer,
+  },
+};
```

Comment:
the shim could use existing Node crypto implementation if you want results that look more like the originals

```suggestion
const nodeCrypto = require('crypto');
global.crypto = {
  getRandomValues: () => {
    return nodeCrypto.randomFillSync(buffer);
  },
  subtle: {
    digest: (algorithm, data) => {
      const hash = nodeCrypto.createHash(algorithm);
      return hash.update(data).digest();
  },
};
```
^ warning: may contain typos

Reply:
OR
this could be stubbed with sinon in a before or beforeEach and random values would no longer be random so they could be used in assertions

Reply:
Getting error `Digest method not supported` :/